app/src/main/java/org/mozilla/fenix/components/toolbar/BrowserToolbarController.kt

```diff
@@ -340,6 +347,7 @@ class DefaultBrowserToolbarController(
                 Event.BrowserMenuItemTapped.Item.READER_MODE_APPEARANCE
             ToolbarMenu.Item.OpenInApp -> Event.BrowserMenuItemTapped.Item.OPEN_IN_APP
             ToolbarMenu.Item.Bookmark -> Event.BrowserMenuItemTapped.Item.BOOKMARK
+            ToolbarMenu.Item.AddonsManager -> Event.BrowserMenuItemTapped.Item.BOOKMARK // FIXME
```

Comment:
@boek @gabrielluong I think we need this updated and a data review?

Reply:
No worries, I already fixed this. Haven't pushed the latest yet.